### PR TITLE
Basic Tests

### LINKED ISSUE #132: Initial Setup and Inclusion of Basic Unit Tests
Basic unit tests need to be written in order to provide a base for further testing of the project.

### PR DESCRIPTION
**Pull request by DanielDowling999**
Basic unit tests of components and the main app page. Does not cover a lot of the project yet, but once I'm able to understand how to run tests with our private-router, more will be included.

Closes #132

### CODE REVIEW

File: src/components/Navbar.jsx [hunk 1]
```diff
@@ -23,12 +23,14 @@ function Navbar() {
       <div>
         <Link to="/home" className="button-link">
           <div className="navbar__icon__group">
+            <strong data-testid="home"/>
```
Comment (franc-hoon) [inline src/components/Navbar.jsx]: what are these in a strong tag? I think you can just add it to the div. Also make the id's a bit more specific, e.g `nav-home-icon` or something like that

File: src/tests/components/Task.test.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+test("Renders Task and Subtask correctly", () => {
+    const{container} = render(<BrowserRouter><Task expanded checked name = "Test 1" subtasks =  {[{id: 0, name: 'Subtask'}]}/></BrowserRouter>);
+    expect(container).toHaveTextContent("Test 1")
+    expect(container).toHaveTextContent("Subtask")
+})
+
+
+test("Renders multiple subtasks correctly", () =>  {
+    const{container} = render(<BrowserRouter><Task expaned name = "Test 2" subtasks = {[{id: 0, name: "Subtask 1"}, {id:1, name: "Subtask 2"},
```
Comment (franc-hoon) [inline src/tests/components/Task.test.jsx]: it's better to have the mock data in a variable for readability.

**Review by franc-hoon — CHANGES_REQUESTED** [review]

File: src/components/Navbar.jsx (hunk 1 above)
Comment (DanielDowling999) [reply src/components/Navbar.jsx]: Ah, the strong tag seems to be unnecessary. The tutorial I followed simply used it. I'll change that and make the id's more specific!

File: src/tests/components/Task.test.jsx (hunk 2 above)
Comment (DanielDowling999) [reply src/tests/components/Task.test.jsx]: Fixed!

**Review by franc-hoon — APPROVED** [review]
lgtm! thanks for doing boring testing :) 
try to run prettier before committing

**Review by lucas2005gao — APPROVED** [review]
Looks good!  Let me know when you got time tomorrow morning and let's run prettier before merging this!

**Pull request merged**